src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveforbarn/GjeldendeBarn.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.familie.ef.sak.iverksett.oppgaveforbarn
+
+import nonapi.io.github.classgraph.json.Id
+import java.time.LocalDate
+import java.util.UUID
+
+data class GjeldendeBarn(@Id
```

Comment:
Usikker på om jeg kommer til å skjønne dette klassenavnet om 2 år, men har ikke noe supert forslag for hånden - @blommish ?

Reply:
`BarnTilUtpukkForOppgave` eller noe slik?